apps/ledger-live-mobile/src/components/RootNavigator/WalletTabNavigator.tsx

```diff
@@ -34,50 +36,55 @@ export default function WalletTabNavigator() {
   const { t } = useTranslation();
 
   return (
-    // <Box bg={"red"} flex={1}>
-    <WalletTab.Navigator
-      initialRouteName={lastVisitedTab}
-      tabBar={tabBarOptions}
-      style={{ backgroundColor: "transparent" }}
-      sceneContainerStyle={{ backgroundColor: "transparent" }}
-      tabBarOptions={{ style: { backgroundColor: "transparent" } }}
-      screenOptions={{
-        lazy: true,
-      }}
-      screenListeners={{
-        state: (e: NavigationContainerEventMap["state"]) => {
-          if (
-            e?.data?.state?.routeNames &&
-            (e?.data?.state?.index || e?.data?.state?.index === 0)
-          ) {
-            dispatch(
-              setWalletTabNavigatorLastVisitedTab(
-                e.data.state.routeNames[e.data.state.index],
-              ),
-            );
-          }
-        },
-      }}
-    >
-      <WalletTab.Screen
-        name={ScreenName.Portfolio}
-        component={
-          readOnlyModeEnabled && accounts.length <= 0
-            ? ReadOnlyPortfolio
-            : Portfolio
-        }
-        options={{
-          title: t("wallet.tabs.crypto"),
+    <>
+      {/* <Box bg={"red"} flex={1}> */}
```

Comment:
We can remove that no?

Reply:
i left this because i find some nice thing for the later animation i was afraid to forgot but yes i will remove

Reply:
Ok no problem :)